.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# No code owners for all files
+*
```

Comment:
I tested in a personal repo, and wildcard without code owners here means that paths other than `web/frontend` will not automatically add me as a reviewer.